torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -2279,6 +2279,10 @@ struct to_ir {
         return aten::__not__;
       case TK_FLOOR_DIV:
         return aten::floordiv;
+      case TK_LSHIFT:
+        return aten::leftshift;
```

Comment:
Just to keep the names consistent with the others/Python can we have these be `aten::__lshift__` and `aten::__rshift__`?